DEVELOPER.md

```diff
@@ -109,61 +186,34 @@ docker exec -u vscode -w /workspaces/terraform-provider-power-platform -it <your
 1. `cd` to a parent folder where main.tf exists
 1. Run `terraform apply`
 
-## Running Acceptance Tests
-
-To run all acceptance tests
-
-```bash
-make acctest
-```
-
-To run single acceptance test
-
-```bash
-TF_ACC=1 go test -v ./... -run TestAcc<test_name>
-```
-
-## Running Unit Tests
-
-To run all unit tests
-
-```bash
-make unittest
-```
-
-To run single unit test
+## Testing Guidelines
 
-```bash
-TF_ACC=0 go test -v ./... -run TestUnit<test_name>
-```
-
-> [!NOTE]
-> The tests require permissions on the folders, these permissions are assigned when creating your container. If you have permission problems when running the unit tests, you can rebuild your development container or run the following commands again to assign the permissions to the necessary folders.
+Quality tests are essential to ensure our provider works reliably across different environments and scenarios. As a contributor, you'll be expected to write comprehensive tests for any code you submit.
 
-```bash
-sudo chown -R vscode /workspaces/terraform-provider-power-platform/
-sudo chown -R vscode /go/pkg
-```
+Our testing approach includes both unit tests and acceptance tests:
 
-## Writing Tests
+- **Unit tests**: Fast-executing tests that verify individual components in isolation
+- **Acceptance tests**: End-to-end tests that validate provider behavior against actual Power Platform APIs
```

Comment:
we recommend that every acc test has its unit test equivalent. There are exceptions but, that would be a general rule